elements/pfe-cta/src/pfe-cta--lightdom.scss

```
@@ -96,15 +96,20 @@ $LOCAL: cta;
 
 @media screen and (-ms-high-contrast: active), screen and (-ms-high-contrast: none) { //ie11
   pfe-cta.PFElement{
-    &[pfe-priority="primary"] { 
+    &[pfe-priority="primary"] {
       a, a:visited, a:focus, a:hover {
         color:  #{pfe-fallback(ui-link--on-saturated)} !important;
       }
     }
-    &[pfe-priority="primary"][pfe-color="lightest"] { 
+    &[pfe-priority="primary"][pfe-color="lightest"] {
+      a, a:visited, a:focus, a:hover {
+        color:  #{pfe-fallback(ui-link)} !important;
+      }
+    }
+    &[pfe-priority="secondary"][pfe-variant="wind"] {
```

Since this is the same as the code for `&[pfe-priority="primary"][pfe-color="lightest"]`, add this to the stack above.
```suggestion
    &[pfe-priority="primary"][pfe-color="lightest"],
    &[pfe-priority="secondary"][pfe-variant="wind"] {
      a, a:visited, a:focus, a:hover {
        color:  #{pfe-fallback(ui-link)} !important;
      }
    }
```